examples/structured_data/collaborative_filtering_movielens.py

```diff
@@ -0,0 +1,217 @@
+"""
+Title: Collaborative Filtering for Movie Recommendations
+Author: Siddhartha Banerjee
+Date created: 2020/05/24
+Last modified: 2020/05/24
+Description: Recommending movies using user and movie embeddings trained using Movielens small dataset.
+"""
+"""
+## Introduction
+
+This example looks at
+[Collaborative filtering using Movielens dataset](https://www.kaggle.com/c/movielens-100k)
+to recommend movies to users.
```

Comment:
Please add a quick explanation of the model, and maybe links to external references about collaborative filtering.

Reply:
Done!